build/getDependencies-windows.sh

```diff
@@ -132,6 +132,7 @@ mkdir -p ../DistFiles/pdf
 mkdir -p ../Downloads
 mkdir -p ../MercurialExtensions
 mkdir -p ../MercurialExtensions/fixutf8
+mkdir -p ../build
 mkdir -p ../build/
```

Comment:
I agree. Doesn't make a difference to me. But it's generated code, so there's not much we can do about that other than debugging the BuildDependency tool - and I'm not sure that this is worth it.